telegram/ext/jobqueue.py

```diff
@@ -21,10 +21,22 @@
 import logging
 import time
 import warnings
+import datetime
+from enum import Enum
```

Comment:
Enums are a new feature that appeared in 3.4. Note that this library also supports python 2.7.
While they have been backported and are available on pypi, I would rather not add a whole dependency for something that is used so very little.